src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -97,7 +97,7 @@ public function dump($input, int $inline = 0, int $indent = 0, int $flags = 0):
                 if ($value instanceof TaggedValue) {
                     $output .= sprintf('%s%s !%s', $prefix, $dumpAsMap ? Inline::dump($key, $flags).':' : '-', $value->getTag());
 
-                    if (Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK & $flags && \is_string($value->getValue()) && false !== strpos($value->getValue(), "\n") && false === strpos($value->getValue(), "\r\n")) {
+                    if (Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK & $flags && \is_string($value->getValue()) && false !== strpos($value->getValue(), "\n") && false === strpos($value->getValue(), "\r")) {
```

Comment:
What is the reason for this change?

Reply:
@xabbuh I wanted to match your change from https://github.com/symfony/symfony/commit/da7870433f0567cde7371d66192103d417e99661.

The idea is to have a consistent logic regardless of TaggedValue or not.

Reply:
Ah I see, makes sense to me. Can we also add a test case for that?

Reply:
@xabbuh sure thing, done 😄